include/core/jbackend-operation.h

```diff
@@ -108,10 +108,29 @@ gboolean j_backend_operation_unwrap_db_update (JBackend*, gpointer, JBackendOper
 gboolean j_backend_operation_unwrap_db_delete (JBackend*, gpointer, JBackendOperation*);
 gboolean j_backend_operation_unwrap_db_query (JBackend*, gpointer, JBackendOperation*);
 
+gboolean j_backend_operation_unwrap_reset(JBackend* backend,gpointer,JBackendOperation*);
+
 gboolean j_backend_operation_to_message (JMessage* message, JBackendOperationParam* data, guint len);
 gboolean j_backend_operation_from_message (JMessage* message, JBackendOperationParam* data, guint len);
 gboolean j_backend_operation_from_message_static (JMessage* message, JBackendOperationParam* data, guint len);
 
+static const JBackendOperation j_backend_operation_reset = {
```

Comment:
Same question about naming. The structs should also be in the same order as the functions.

Reply:
the intention was to have one reset function for all backend types